### PR TITLE
Adding vertnet amphibians script

### PR DESCRIPTION
**Pull request by ShivamNegi**
#819 
@henrykironde I couldn't test this script. As I am unable to add it to the retriever data set list.

### CODE REVIEW

File: scripts/vertnet_amphibians.py [hunk 1]
```diff
@@ -0,0 +1,237 @@
+# -*- coding: latin-1 -*-
+# retriever
```
Comment (henrykironde) [inline scripts/vertnet_amphibians.py]: no space `# retriever` > `#retriever`

Comment (henrykironde) [inline scripts/vertnet_amphibians.py]: That will make it work, and update the version file, let me know if it works

**Comment by henrykironde** [comment]
@ShivamNegi lets make one commit message
 ```
Adding vertnet individual dataset support

Adding vertnet birds script
Adding vertnet fishes script
Adding vertnet reptiles script
Adding vertnet mammals script
Adding vertnet amphibians script
```
Additionally the version numbers for these scripts should be "1.0.0" since they are new scripts
The version file is corrupt. Always note the script changes, in this case the script is corrupted.

**Comment by henrykironde** [comment]
You need to clean your repo.
I suggest running

```
retriever reset all
rm -r scripts
git reset --hard
python version.py
```

This will give you the right version.txt to add and commit

**Comment by ShivamNegi** [comment]
@henrykironde I get this exception on running ```python version.py```
```
➜  retriever git:(vertnet-dataset) ✗ python version.py 
Failed to load script: forest_fires_portugal (scripts)
Exception: Non-ASCII character '\xc3' in file scripts/forest_fires_portugal.py on line 9, but no encoding declared; see http://python.org/dev/peps/pep-0263/ for details (forest_fires_portugal.py, line 9) 
```

**Comment by henrykironde** [comment]
could you give me your system details, I am trying to replicate that error.

**Comment by ShivamNegi** [comment]
HP ProBook 440
8 GB ram, Intel i5 quad core processor
Running Ubuntu 16.04 LTS

running ```python version.py``` produces error in ```Python 2.7.12```. Works fine with ```Python 3.5.2```.

**Comment by ShivamNegi** [comment]
@henrykironde I think this works.

**Comment by henrykironde** [comment]
I tested the installation on windows and it looks like we fail to install. However this does not have anything to do with the scripts. I feel the scripts are fine and ready to go in. It has something to do with the data.

**Comment by ShivamNegi** [comment]
Should multiple tags be added or a single tag is fine?
```self.tags = [u'taxon', u'amphibians']```
I am saying this because all the scripts that are being intialized using ```BasicTextTemplate``` have at least 2 tags.

Also what is the citation for other files? 
Will changing ```[*]``` with ```Aves```, ```Fishes```, ```Mammals```, ```Reptilia``` do?
```
self.citation = "Bloom, D., Wieczorek J., Russell, L. (2016).  VertNet_[Amphibia]_Sept. 2016. CyVerse Data Commons. http://datacommons.cyverse.org/browse/iplant/home/shared/commons_repo/curated/VertNet_[Amphibia]_Sep2016"
```

**Comment by ethanwhite** [comment]
> Should multiple tags be added or a single tag is fine?

A single tag is fine, but it should still be in a list. Multiple tags provides more information so that's why a lot of datasets have them.

> Will changing [*] with Aves, Fishes, Mammals, Reptilia do?

Yes.

**Comment by ShivamNegi** [comment]
@ethanwhite, please review.

**Comment by ethanwhite** [comment]
Looks great. Since @henrykironde's already signed off, in it goes. Thanks!

**Pull request merged**